quantum/via.h

```diff
@@ -85,23 +113,17 @@ enum via_command_id {
     id_unhandled                            = 0xFF,
 };
 
-enum via_keyboard_value_id {
-    id_uptime              = 0x01, //
-    id_layout_options      = 0x02,
-    id_switch_matrix_state = 0x03
-};
+enum via_keyboard_value_id { id_uptime = 0x01, id_layout_options = 0x02, id_switch_matrix_state = 0x03, id_firmware_version = 0x04, id_device_indication = 0x05 };
 
-enum via_lighting_value {
-    // QMK BACKLIGHT
-    id_qmk_backlight_brightness = 0x09,
-    id_qmk_backlight_effect     = 0x0A,
+enum via_channel_id { id_custom_channel = 0, id_qmk_backlight_channel = 1, id_qmk_rgblight_channel = 2, id_qmk_rgb_matrix_channel = 3, id_qmk_audio_channel = 4 };
 
-    // QMK RGBLIGHT
-    id_qmk_rgblight_brightness   = 0x80,
-    id_qmk_rgblight_effect       = 0x81,
-    id_qmk_rgblight_effect_speed = 0x82,
-    id_qmk_rgblight_color        = 0x83,
-};
+enum via_qmk_backlight_value { id_qmk_backlight_brightness = 1, id_qmk_backlight_effect = 2 };
+
+enum via_qmk_rgblight_value { id_qmk_rgblight_brightness = 1, id_qmk_rgblight_effect = 2, id_qmk_rgblight_effect_speed = 3, id_qmk_rgblight_color = 4 };
+
+enum via_qmk_rgb_matrix_value { id_qmk_rgb_matrix_brightness = 1, id_qmk_rgb_matrix_effect = 2, id_qmk_rgb_matrix_effect_speed = 3, id_qmk_rgb_matrix_color = 4 };
+
+enum via_qmk_audio_value { id_qmk_audio_enable = 1, id_qmk_audio_clicky_enable = 2 };
```

Comment:
for better readability.  


(Trailing comma after the last value prevents formatting from putting it all on one line)
```suggestion
enum via_keyboard_value_id {
    id_uptime = 0x01,
    id_layout_options = 0x02,
    id_switch_matrix_state = 0x03,
    id_firmware_version = 0x04,
    id_device_indication = 0x05,
};

enum via_channel_id { 
    id_custom_channel = 0,
    id_qmk_backlight_channel = 1,
    id_qmk_rgblight_channel = 2,
    id_qmk_rgb_matrix_channel = 3,
    id_qmk_audio_channel = 4,

enum via_qmk_backlight_value { 
    id_qmk_backlight_brightness = 1, 
    id_qmk_backlight_effect = 2,
};

enum via_qmk_rgblight_value {
    id_qmk_rgblight_brightness = 1,
    id_qmk_rgblight_effect = 2,
    id_qmk_rgblight_effect_speed = 3,
    id_qmk_rgblight_color = 4,
};

enum via_qmk_rgb_matrix_value {
    id_qmk_rgb_matrix_brightness = 1,
    id_qmk_rgb_matrix_effect = 2,	
    id_qmk_rgb_matrix_effect_speed = 3,
    id_qmk_rgb_matrix_color = 4,
};

enum via_qmk_audio_value {
    id_qmk_audio_enable = 1,
    id_qmk_audio_clicky_enable = 2,
};
```